### DISCUSSION TITLE
有 ERLayout 的使用文档吗？

### DISCUSSION THREAD

**Discussion by LLLaiYoung** [antvis/X6#2455] [Q&A]
各位大佬有 ERLayout 的使用文档吗？或者说有可参考的示例。

感谢各位大佬！

**Comment by zmcode** [comment]
没有看到你说的ERLayout布局, https://x6.antv.vision/zh/examples/showcase/practices#er, 如果你指的是这个, 这里面就有详细的代码了

作者网页提供上的布局方案,都是引入g6的布局代码的, 可以下载g6的代码来看, g6的部分布局也是借鉴了g6内部的[开源算法](https://github.com/antvis/algorithm)以及像斥力的d3.force库来实现的

**Comment by world56** [comment]
https://github.com/antvis/X6/discussions/2495
